invenio_notifications/templates/semantic-ui/invenio_notifications/email/community-submission.submitted.html

```diff
@@ -0,0 +1,7 @@
+<p>{{ _('New community submission') }}</p>
```

Comment:
do you plan to move these templates in the future to invenio-communities?

Reply:
Yup, see: https://github.com/inveniosoftware/invenio-notifications/pull/7#discussion_r1111599913